p2p/helpers.go

```diff
@@ -17,6 +18,9 @@ import (
 	p2p_pb "github.com/celestiaorg/go-header/p2p/pb"
 )
 
+// retries is the amount of attempts to open the stream in case of an error.
+const retries = 5
```

Comment:
Can we make this configurable now?

Reply:
you want to make a config param for it rn?

Reply:
I'd prefer not to do this tbh

Reply:
I agree so that we can trouble shoot and up it to 10 in the case we need to do this.